nfs_types.go

```diff
@@ -103,23 +104,30 @@ type NFSExportModify struct {
 	// NFS enforced security type for users accessing an NFS Export.
 	MinSecurity string `json:"min_security,omitempty"`
 	// Hosts with no access to the NFS export or its snapshots.
-	NoAccessHosts []string `json:"no_access_hosts,omitempty"`
+	// Empty list is a valid value, so omitempty should not be used.
+	NoAccessHosts []string `json:"no_access_hosts"`
```

Comment:
Hi,
For all the hosts list, there are 'add' and 'remove' lists in _NFSExportModify_. Please make use of those fields to add or remove a host from the list during modify operation.
Removing _omitempty_ for these lists have an effect on other Client library users.